src/components/CardDropdownButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
This is really cool. Thanks for abstracting this and making it a reusable component